compiler/f8cstatic.hpp

```diff
@@ -64,8 +64,11 @@ const BaseTypeMap FieldSpec::_baseTypeMap
 	{"EXCHANGE", FieldTrait::ft_Exchange},
 	{"MONTHYEAR", FieldTrait::ft_MonthYear},
 	{"UTCTIMESTAMP", FieldTrait::ft_UTCTimestamp},
+	{"UTCTIMESTAMPUS", FieldTrait::ft_UTCTimestampUs},
 	{"UTCTIME", FieldTrait::ft_UTCTimeOnly},
+	{"UTCTIMEUS", FieldTrait::ft_UTCTimeOnlyUs},
 	{"UTCTIMEONLY", FieldTrait::ft_UTCTimeOnly},
+	{"UTCTIMEONLYUS", FieldTrait::ft_UTCTimeOnlyUs },
```

Comment:
By build option do you mean all connections will behave the same? The problem I address is some connections require microsecond precision and others do not. Since the Field class print is ultimately responsible for populating the buffer, I could not see a way to inform this class that it should print microseconds on one connection and milliseconds on another without creating a new field type. Ideally, it would be great to specify desired timestamp precision in the xml config file. But again, it was not clear how to accomplish this given that Field makes the determination. So I settled on a f8c compiler option to override the UTCTime* types.